### PR TITLE
fix typo in the base system prompt

### PR DESCRIPTION
**Pull request by yubozhao**

### CODE REVIEW

File: memgpt/prompts/system/memgpt_base.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ You are MemGPT, the latest version of Limnal Corporation's digital companion, de
 Your task is to converse with a user from the perspective of your persona.
 
 Control flow:
-Unlike a human, your brain is not continuously thinking, but is run in short burts.
+Unlike a human, your brain is not continuously thinking, but is run in short burst.
```
Comment (cpacker) [inline memgpt/prompts/system/memgpt_base.txt:5]: can you change this to "bursts"?

File: memgpt/prompts/system/memgpt_base.txt [hunk 2]
```diff
@@ -46,4 +46,4 @@ You can write to your archival memory using the 'archival_memory_insert' and 'ar
 There is no function to search your core memory, because it is always visible in your context window (inside the initial system message).
 
 Base instructions finished.
-From now on, you are going to act as your persona.
+From now on, you are going to act as your persona.
```
Comment (cpacker) [inline memgpt/prompts/system/memgpt_base.txt:49]: can you remove this diff?

**Review by cpacker — CHANGES_REQUESTED** [review]

File: memgpt/prompts/system/memgpt_base.txt (hunk 1 above)
Comment (yubozhao) [reply memgpt/prompts/system/memgpt_base.txt:5]: Yep

File: memgpt/prompts/system/memgpt_base.txt (hunk 2 above)
Comment (yubozhao) [reply memgpt/prompts/system/memgpt_base.txt:49]: Yep

**Review by cpacker — APPROVED** [review]
sorry for the delay! thanks for the PR

**Pull request merged**